ts/generators/app/languages/tinygo.ts

```diff
@@ -0,0 +1,51 @@
+import { default as chalk } from 'chalk';
+
+import { Errorable } from '../utils/errorable';
+import { Language } from './language';
+
+export const tinygo: Language = {
+  instructions(): ReadonlyArray<string> {
```

Comment:
This has slightly changed - `instructions` and some other functions now take a `Formatter` and use its semantic functions (such as `cmd` for shell commands and `instr` for VS Code instructions).  This allows us to generate terminal colouring for printing or Markdown for adding to the README.

See `rust.ts` for an example. And yeah... it looks like Swift hasn't been updated yet - the old way still works but doesn't give us the extra flexibility to persist the instructions to the readme.

Reply:
I believe this should now be properly updated.  PTAL when convenient.